src/Workspaces/Core/Portable/FindSymbols/FindReferences/DependentProjectsFinder.cs

```diff
@@ -364,6 +364,11 @@ private static Lazy<HashSet<string>> CreateInternalsVisibleToMap(IAssemblySymbol
                     }
 
                     var value = (string)typeNameConstant.Value;
+                    if (string.IsNullOrWhiteSpace(value))
```

Comment:
The nittiest of nits: Is there any way to have meaningful whitespace here? If the string is ```"<whitespace>,<other stuff>"``` then we'd previously let that ```<whitespace>``` get into the map.